webapp/portlet/src/main/resources/locale/portlet/social/ProfileContactInformation_en.properties

```diff
@@ -25,3 +25,5 @@ profileContactInformation.team=Team
 profileContactInformation.profession=Profession
 profileContactInformation.country=Country
 profileContactInformation.city=City
+profileContactInformation.error.invalidField=Only alpha, digit, dash and underscore characters allowed for the field "{0}"
```

Comment:
Please avoid using double quotes

Reply:
Okay but that is how the message is set.